### PR TITLE
README for Verrazzano distribution

### PR DESCRIPTION
**Pull request by bhatpmk**
Includes initial version of readme for Verrazzano release distributions

### CODE REVIEW

File: release/docs/README_COMMERCIAL.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+     * `charts/verrazzano-platform-operator/`: Helm chart for Verrazzano Platform Operator       
+     * `profiles/<default.yaml, dev.yaml, managed-cluster.yaml, oci.yaml, ocne.yaml>`: Sample built-in configuration profiles to simplify a Verrazzano installation        
+     * `verrazzano-bom.json`: Bill of materials (BOM) containing the list of docker images for Verrazzano and the components installed by Verrazzano       
+  * `images/`:  Verrazzano Enterprise Container Platform archives for private registry install    
+
+## Install Verrazzano using a private container registry
+
+You can install Verrazzano using a private Docker-compliant container registry. This requires the following:
+
+*    Load the product images from `/images` into your private registry Loading all required Verrazzano container images from `/images` into your own registry and repository.
```
Comment (rosemarymarano) [inline release/docs/README_COMMERCIAL.md:71]: Load the product images from `/images` into your private registry Loading all required Verrazzano container images from `/images` into your own registry and repository. -> (Not sure what to make of this "sentence" but I'll take a stab at re-writing it. If it is not what you want it to state, please correct it) -> Load the product images from `/images` into your private registry. This includes loading all the required Verrazzano container images from `/images` into your own registry and repository.

File: release/docs/README_COMMERCIAL.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+*    Install Verrazzano using the installation instructions, see the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).
```
Comment (rosemarymarano) [inline release/docs/README_COMMERCIAL.md:72]: Install Verrazzano using the installation instructions, see the -> Install Verrazzano using the instructions in the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).

File: release/docs/README_COMMERCIAL.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## Install Verrazzano using a private container registry
+
+You can install Verrazzano using a private Docker-compliant container registry. This requires the following:
+
+*    Load the product images from `/images` into your private registry Loading all required Verrazzano container images from `/images` into your own registry and repository.
+*    Install Verrazzano using the installation instructions, see the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).     
+
+## Support
+
+*    If you have any questions about Verrazzano Commercial Edition, please join our [slack channel](https://bit.ly/3gOeRJn).
```
Comment (rosemarymarano) [inline release/docs/README_COMMERCIAL.md:76]: slack  -> Slack (product name)

File: release/docs/README_OPEN_SOURCE.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
+# Verrazzano Open Source Edition
+
+Verrazzano open source edition is a fully featured enterprise container platform for deploying cloud-native and traditional applications in multicloud
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:3]: Verrazzano open source edition -> Verrazzano Open Source Edition 
fully featured -> fully-featured
cloud-native -> cloud native

File: release/docs/README_OPEN_SOURCE.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
+# Verrazzano Open Source Edition
+
+Verrazzano open source edition is a fully featured enterprise container platform for deploying cloud-native and traditional applications in multicloud 
+and hybrid environments. It is a freely available, community supported, open source distribution of Verrazzano Enterprise Container Platform.
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:4]: of Verrazzano Enterprise Container Platform -> of the Verrazzano Enterprise Container Platform

File: release/docs/README_OPEN_SOURCE.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+# Overview
+Verrazzano open source edition includes the following capabilities:
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:7]: Verrazzano open source edition -> Verrazzano Open Source Editio

File: release/docs/README_OPEN_SOURCE.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+- Hybrid and multicluster workload management
+- Special handling for WebLogic, Coherence, and Helidon applications
+- Multicluster infrastructure management
+- Integrated and pre-wired application monitoring
+- Integrated security
+- DevOps and GitOps enablement
+
+# Components
+Verrazzano open source edition includes a curated set of open source components – many that you may already use and trust,
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:17]: Verrazzano open source edition -> Verrazzano Open Source Edition

File: release/docs/README_OPEN_SOURCE.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+| OpenSearch                   | 1.2.3   | Provides a distributed, multitenant-capable full-text search engine.                     |
+| OpenSearch Dashboards        | 1.2.0   | Provides search and data visualization capabilities for data indexed in OpenSearch.      |
+| Prometheus                   | 2.34.0  | Provides event monitoring and alerting.                                                  |
+| Prometheus Adapter           | 0.9.1   | Provides metrics in support of pod autoscaling.                                          |
+| Prometheus Operator          | 0.55.1  | Provides management for Prometheus monitoring tools.                                     |
+| Prometheus Pushgateway       | 1.4.2   | Allows ephemeral and batch jobs to expose their metrics to Prometheus.                   |
+| Rancher                      | 2.6.7   | Manages multiple Kubernetes clusters.                                                    |
+| WebLogic Kubernetes Operator | 3.4.0   | Assists with deploying and managing WebLogic domains.                                    |
+
+## Distribution Layout
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:46]: Distribution Layout -> Distribution layout

File: release/docs/README_OPEN_SOURCE.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+Verrazzano Community Edition includes the following artifacts:
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:48]: Verrazzano Community Edition ->  Verrazzano Open Source Edition

File: release/docs/README_OPEN_SOURCE.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+* `operator.yaml`: a collection of Kubernetes manifests that can be used to deploy the Verrazzano platform operator
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:50]: Capitalize the first letter of the first word in each of the following descriptive sentences and put a period at the end of each one.

File: release/docs/README_OPEN_SOURCE.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+* `operator.yaml.sha256`: sha256 checksum of the operator.yaml artifact
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:51]: operator.yaml -> `operator.yaml` (code font)

File: release/docs/README_OPEN_SOURCE.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+* `verrazzano-<major>.<minor>.<patch>-linux-amd64.tar.gz`: Verrazzano distribution with binaries compiled for AMD64 Linux Distributions
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:52]: Distributions -> distributions

File: release/docs/README_OPEN_SOURCE.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+* `verrazzano-<major>.<minor>.<patch>-linux-amd64.tar.gz.sha256`: SHA256 checksum of `verrazzano-<major>.<minor>.<patch>-linux-amd64.tar.gz` artifact
+* `verrazzano-<major>.<minor>.<patch>-darwin-amd64.tar.gz`: Verrazzano distribution with binaries compiled for Darwin AMD64 (MacOS) Distributions
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:54]: Distributions -> distributions

File: release/docs/README_OPEN_SOURCE.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+* `verrazzano-<major>.<minor>.<patch>-darwin-amd64.tar.gz.sha256`: SHA256 checksum of `verrazzano-<major>.<minor>.<patch>-darwin-amd64.tar.gz` artifact
+
+The layout of the Verrazzano distribution is given :
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:57]: is given : -> is as follows:

File: release/docs/README_OPEN_SOURCE.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+
+* `verrazzano-<major>.<minor>.<patch>/`
+  * `README.md`
+  * `README.html` 
+  * `LICENSE`: The Universal Permissive License (UPL)
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:62]: Please apply the same edits as I described in the README_COMMERCIAL.md

**Comment by bhatpmk** [comment]
VZ-6863, VZ-6902

File: release/docs/README_OPEN_SOURCE.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+     * `k8s/verrazzano-platform-operator.yaml`: Collection of Kubernetes manifests to deploy the Verrazzano platform operator   
+     * `charts/verrazzano-platform-operator/`: Helm charts for Verrazzano Platform Operator       
+     * `profiles/<default.yaml, dev.yaml, managed-cluster.yaml, oci.yaml, ocne.yaml>`: Sample built-in configuration profiles to simplify a Verrazzano installation        
+     * `verrazzano-bom.json`: Bill of materials (BOM) containing the list of docker images required for installing Verrazzano          
+
+## Install Verrazzano using a private container registry
+
+You can install Verrazzano using a private Docker-compliant container registry. This requires the following:
+
+*    Loading all required Verrazzano container images into your own registry and repository.
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:76]: all required -> all the required

File: release/docs/README_OPEN_SOURCE.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+*    Installing the Verrazzano platform operator with the private registry and repository used to load the images.
+
+To obtain the required Verrazzano images and install from your private registry, you must:
+*    Download the required Verrazzano images defined in `verrazzano-bom.json` as tar.gz file using `vz-registry-image-helper.sh`
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:80]: as tar.gz -> as `tar.gz` (code font)
Please put a period at the end of each sentence.

**Review by rosemarymarano — COMMENTED** [review]
Some edits and suggestions.

File: release/docs/README_COMMERCIAL.md [hunk 18]
```diff
@@ -0,0 +1,80 @@
...
+*    For instructions on using Verrazzano, see the [Verrazzano documentation](https://verrazzano.io/latest/docs/).
+*    For detailed installation instructions, see the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).
+*    More information about troubleshooting is available [here](https://verrazzano.io/latest/docs/troubleshooting/diagnostictools/).
+*    Please submit [bugs or enhancements requests](https://github.com/verrazzano/verrazzano/issues/new/choose) in GitHub.
```
Comment (MelwynPaul) [inline release/docs/README_COMMERCIAL.md:80]: From: Please submit [bugs or enhancements requests](https://github.com/verrazzano/verrazzano/issues/new/choose) in GitHub.
To: To report a bug or request for an enhancement, submit them through [bugs or enhancements requests](https://github.com/verrazzano/verrazzano/issues/new/choose) in GitHub.

File: release/docs/README_OPEN_SOURCE.md [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+*    Download the required Verrazzano images defined in `verrazzano-bom.json` as tar.gz file using `vz-registry-image-helper.sh`       
+     ```
+     $ bin/vz-registry-image-helper.sh -b ../manifests/verrazzano-bom.json -f ~/verrazzano_distribution.tar.gz
+     ```
+*    Expand the TAR file to a directory of your choice
+     ```
+     $ tar xvf ~/verrazzano_distribution.tar.gz -C <directory>
+     ```
+*    Load the product images into your private registry    
+*    Install Verrazzano using the installation instructions, see the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).
```
Comment (MelwynPaul) [inline release/docs/README_OPEN_SOURCE.md:89]: From: Install Verrazzano using the installation instructions, see the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/). 
To: Install Verrazzano using the instructions in the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).

**Review by MelwynPaul — COMMENTED** [review]
Few edits and suggestions.

File: release/docs/README_COMMERCIAL.md [hunk 20]
```diff
@@ -0,0 +1,80 @@
+# Verrazzano Commercial Edition
+
+Verrazzano Commercial Edition is a fully featured enterprise container platform for deploying cloud-native and traditional applications in multicloud
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply release/docs/README_COMMERCIAL.md]: Updated as per the comment

File: release/docs/README_COMMERCIAL.md [hunk 21]
```diff
@@ -0,0 +1,80 @@
+# Verrazzano Commercial Edition
+
+Verrazzano Commercial Edition is a fully featured enterprise container platform for deploying cloud-native and traditional applications in multicloud 
+and hybrid environments, packaged for the enterprise.
+
+# Overview
+It includes the following capabilities:
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply release/docs/README_COMMERCIAL.md]: Done

File: release/docs/README_COMMERCIAL.md [hunk 22]
```diff
@@ -0,0 +1,80 @@
...
+| Prometheus                   | 2.34.0  | Provides event monitoring and alerting.                                                  |
+| Prometheus Adapter           | 0.9.1   | Provides metrics in support of pod autoscaling.                                          |
+| Prometheus Operator          | 0.55.1  | Provides management for Prometheus monitoring tools.                                     |
+| Prometheus Pushgateway       | 1.4.2   | Allows ephemeral and batch jobs to expose their metrics to Prometheus.                   |
+| Rancher                      | 2.6.7   | Manages multiple Kubernetes clusters.                                                    |
+| WebLogic Kubernetes Operator | 3.4.0   | Assists with deploying and managing WebLogic domains.                                    |
+
+## Distribution Layout
+
+Verrazzano Commercial Edition distribution includes the following artifacts:
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply release/docs/README_COMMERCIAL.md]: Done

File: release/docs/README_COMMERCIAL.md [hunk 23]
```diff
@@ -0,0 +1,80 @@
...
+
+* `verrazzano-<major>.<minor>.<patch>/`
+  * `README.md`
+  * `README.html` 
+  * `LICENSE`: The Universal Permissive License (UPL)
+  * `bin/`    
+     * `linux-amd64/vz`: Verrazzano command line interface for Linux AMD64 architecture
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply release/docs/README_COMMERCIAL.md]: Done

File: release/docs/README_OPEN_SOURCE.md (hunk 13 above)
Comment (bhatpmk) [reply release/docs/README_OPEN_SOURCE.md]: Done

File: release/docs/README_COMMERCIAL.md [hunk 24]
```diff
@@ -0,0 +1,80 @@
...
+  * `bin/`    
+     * `linux-amd64/vz`: Verrazzano command line interface for Linux AMD64 architecture       
+     * `linux-arm64/vz`: Verrazzano command line interface for Linux ARM64 architecture    
+     * `darwin-amd64/vz`: Verrazzano command line interface for Darwin AMD64 architecture       
+     * `darwin-arm64/vz`: Verrazzano command line interface for Darwin ARM64 architecture                   
+     * `vz-registry-image-helper.sh, bom_utils.sh`:  Helper scripts to download the images from the bill of materials (BOM)
+  * `manifests/`     
+     * `k8s/verrazzano-platform-operator.yaml`: Kubernetes manifests to deploy the Verrazzano platform operator   
+     * `charts/verrazzano-platform-operator/`: Helm chart for Verrazzano Platform Operator       
+     * `profiles/<default.yaml, dev.yaml, managed-cluster.yaml, oci.yaml, ocne.yaml>`: Sample built-in configuration profiles to simplify a Verrazzano installation
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply release/docs/README_COMMERCIAL.md]: I have removed the profiles for now, as there is more work required

File: release/docs/README_COMMERCIAL.md (hunk 1 above)
Comment (bhatpmk) [reply release/docs/README_COMMERCIAL.md]: It was my mistake, corrected the text

File: release/docs/README_COMMERCIAL.md [hunk 25]
```diff
@@ -0,0 +1,80 @@
...
+You can install Verrazzano using a private Docker-compliant container registry. This requires the following:
+
+*    Load the product images from `/images` into your private registry Loading all required Verrazzano container images from `/images` into your own registry and repository.
+*    Install Verrazzano using the installation instructions, see the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).     
+
+## Support
+
+*    If you have any questions about Verrazzano Commercial Edition, please join our [slack channel](https://bit.ly/3gOeRJn).
+*    For instructions on using Verrazzano, see the [Verrazzano documentation](https://verrazzano.io/latest/docs/).
+*    For detailed installation instructions, see the [Verrazzano Installation Guide](https://verrazzano.io/latest/docs/setup/install/installation/).
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply release/docs/README_COMMERCIAL.md]: Minor edit to Melwyn's suggestion -> see the [Install Guide](https://verrazzano.io/latest/docs/setup/install/installation/) in the Verrazzano documentation

File: release/docs/README_OPEN_SOURCE.md [hunk 26]
```diff
@@ -43,50 +43,49 @@ and some that were written specifically to pull together all the pieces to make
 | Rancher                      | 2.6.7   | Manages multiple Kubernetes clusters.                                                    |
 | WebLogic Kubernetes Operator | 3.4.0   | Assists with deploying and managing WebLogic domains.                                    |
 
-## Distribution Layout
+## Distribution layout
 
-Verrazzano Community Edition includes the following artifacts:
+Verrazzano Open Source Edition includes the following artifacts:
```
Comment (rosemarymarano) [inline release/docs/README_OPEN_SOURCE.md:48]: Please put a period at the end of each of the following descriptive sentences in this list.

File: release/docs/README_COMMERCIAL.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (bhatpmk) [reply release/docs/README_COMMERCIAL.md]: Done

File: release/docs/README_OPEN_SOURCE.md (hunk 26 above)
Comment (bhatpmk) [reply release/docs/README_OPEN_SOURCE.md:48]: Done

**Review by MelwynPaul — APPROVED** [review]

**Review by agv426 — APPROVED** [review]

**Pull request merged**